### PR TITLE
Debug errors in BadIP 

### PR DESCRIPTION
**Pull request by nitin-techie**
BadIP was initially making decision based on just 1 category listed for the IOC. Also, the category was not displayed.
Customized the code to consider all the categories marked and display result that includes list of all the categories the IP was reported under.

## Description ##
BadIP was initially making decision based on just 1 category listed for the IOC. Also, the category was not displayed.
Customized the code to consider all the categories marked and display result that includes list of all the categories the IP was reported under.

## Does this fix a known existing bug under Issues? ##
Issue #56 

## Type of Change ##
Please delete any options that **do not** apply here:
- [ ] Bug Fix

## Any further info related to the addition ##
Provide any additional details that seem relevant here if necessary, otherwise either delete or leave this section blank.

### CODE REVIEW

**Comment by nitin-techie** [comment]
@TheresAFewConors -- Pls review and if you're okay with the changes, You can merge the latest commit of Sooty.py.

File: Sooty.py [hunk 1]
```diff
@@ -426,10 +426,18 @@ def repChecker():
             response = requests.get(BAD_IPS_URL)
             if response.status_code == 200:
                 result = response.json()
-
-                sc = result['Score']['ssh']
+                sc = 0
+                for each in result['Categories']:
+                    temp = each
+                    sc += result['Score']['temp']
```
Comment (nitin-techie) [inline Sooty.py:432]: Calculate the score for all the categories it was reported under and sum it all.

File: Sooty.py [hunk 2]
```diff
@@ -426,10 +426,18 @@ def repChecker():
...
                 print("  " + str(result['suc']))
+                print("  Total Reports :  " + str(result['ReporterCount']['sum']))
                 print("  Score: " + str(sc))
+                if sc == 0:
+                    print('  Score 0 indicates, this might be a false positive')
+                print("\n IP reported in following Categories :\n")
+                for each in result['Categories']:
+                    print(" *. " + each)
```
Comment (nitin-techie) [inline Sooty.py:440]: Print all the categories name it was reported under

**Review by TheresAFewConors — COMMENTED** [review]
looks like there's an error on the sc value - looking into it

**Comment by TheresAFewConors** [comment]
looks like on BadIP's all the scores are set to 0 from the small subset I looked at. Rewrote a section to skip the BadIP value, (Total Reports provides a good metric), and also added the timestamps of last report.

**Comment by nitin-techie** [comment]
> looks like on BadIP's all the scores are set to 0 from the small subset I looked at. Rewrote a section to skip the BadIP value, (Total Reports provides a good metric), and also added the timestamps of last report.

Some scores are definitely greater than 0.
Besides, if you would'd like to skip that part. I am okay with it.

**Comment by TheresAFewConors** [comment]
> 
> 
> > looks like on BadIP's all the scores are set to 0 from the small subset I looked at. Rewrote a section to skip the BadIP value, (Total Reports provides a good metric), and also added the timestamps of last report.
> 
> Some scores are definitely greater than 0.
> Besides, if you would'd like to skip that part. I am okay with it.

That is quite likely, I only checked 5 or 6 of the more recent reports. 

I just pushed the fix, I like what you done with the specific categories being reported as well, very informative and useful information, I just added the associated timestamp for each category too.

**Comment by nitin-techie** [comment]
> > > looks like on BadIP's all the scores are set to 0 from the small subset I looked at. Rewrote a section to skip the BadIP value, (Total Reports provides a good metric), and also added the timestamps of last report.
> > 
> > 
> > Some scores are definitely greater than 0.
> > Besides, if you would'd like to skip that part. I am okay with it.
> 
> That is quite likely, I only checked 5 or 6 of the more recent reports.
> 
> I just pushed the fix, I like what you done with the specific categories being reported as well, very informative and useful information, I just added the associated timestamp for each category too.

#

> > > looks like on BadIP's all the scores are set to 0 from the small subset I looked at. Rewrote a section to skip the BadIP value, (Total Reports provides a good metric), and also added the timestamps of last report.
> > 
> > 
> > Some scores are definitely greater than 0.
> > Besides, if you would'd like to skip that part. I am okay with it.
> 
> That is quite likely, I only checked 5 or 6 of the more recent reports.
> 
> I just pushed the fix, I like what you done with the specific categories being reported as well, very informative and useful information, I just added the associated timestamp for each category too.

Thanks.
Can i be added to the contributors list ?

**Comment by TheresAFewConors** [comment]
> Thanks.
> Can i be added to the contributors list ?

Of course, I am just waiting for the automated checks to complete first to make sure there are no issues. I will update the list then afterwards.

**Comment by nitin-techie** [comment]
> > Thanks.
> > Can i be added to the contributors list ?
> 
> Of course, I am just waiting for the automated checks to complete first to make sure there are no issues. I will update the list then afterwards.

- [ ] @TheresAFewConors -- Thanks, i am gonna be opening a couple of more issues, which i have already sorted in my local file.

**Pull request merged**